cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,19 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+		Example: `# Create a Cluster Workflow Template with Default Options:
+		argo cluster-template create FILE1.yaml
+
+		# Create a Cluster Workflow Template in JSON Format:
+		argo cluster-template create FILE1.yaml --output json 
+
+		# Create a Cluster Workflow Template with Relaxed Validation (default true):
+		argo cluster-template create FILE1.yaml --strict=false
+
+		# Create a Cluster Workflow Template and Save Output to a File (One of: name|json|yaml|wide):
```

Comment:
```suggestion
		# Create a Cluster Workflow Template and save the output to a JSON file:
```

- the options already list the enumeration, so we don't need to repeat them
- same as above re: capitalization

Reply:
Also this example should be moved to be below the other JSON example so that similar examples are grouped together